ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/serde/json/ValueSpecJsonSerdeSupplierTest.java

```diff
@@ -72,32 +68,4 @@ public void shouldDeserializeDecimalsWithoutStrippingTrailingZeros_Plain() {
     // Then:
     assertThat(result, is(new BigDecimal("10.0")));
   }
-
-  @Test
-  public void shouldSerializeDecimalsWithOutStrippingTrailingZeros_Sr() throws Exception {
-    // Given:
-    final Serializer<Object> serializer = srSerde.getSerializer(srClient, false);
-
-    // When:
-    final byte[] bytes = serializer.serialize("t", new BigDecimal("10.0"));
-
-    // Then:
-    assertThat(JsonSerdeUtils.readJsonSR(bytes, TestJsonMapper.INSTANCE.get(), String.class), is("10.0"));
-  }
-
-  @Test
-  public void shouldDeserializeDecimalsWithoutStrippingTrailingZeros_Sr() {
-    // Given:
-    final Deserializer<Object> deserializer = srSerde.getDeserializer(srClient, false);
-
-    final byte[] jsonBytes = "10.0".getBytes(UTF_8);
-    final byte[] bytes = new byte[jsonBytes.length + JsonSerdeUtils.SIZE_OF_SR_PREFIX];
-    System.arraycopy(jsonBytes, 0, bytes, JsonSerdeUtils.SIZE_OF_SR_PREFIX, jsonBytes.length);
-
-    // When:
-    final Object result = deserializer.deserialize("t", bytes);
-
-    // Then:
-    assertThat(result, is(new BigDecimal("10.0")));
-  }
 }
```

Comment:
Should they be move to somewhere else instead of deleted?

Reply:
I moved it to `ValueSpecJsonSchemaSerdeSupplierTest`